src/app/medInria/medApplication.cpp

```diff
@@ -146,6 +170,9 @@ void medApplication::setMainWindow(medMainWindow *mw)
     QVariant var = QVariant::fromValue((QObject*)d->mainWindow);
     this->setProperty("MainWindow",var);
     d->systemOpenInstructions.clear();
+
+    // Wait until the app is displayed to close itself
+    d->splashScreen->finish(d->mainWindow); 
```

Comment:
The `close` function will be called on the splash screen, but this does not delete the widget unless the attribute `Qt::WA_DeleteOnClose` is set.

Reply:
Indeed, i added a line and rebased the PR